lib/taskana-core/src/main/java/pro/taskana/classification/api/models/Classification.java

```diff
@@ -8,143 +8,146 @@
 public interface Classification extends ClassificationSummary {
 
   /**
-   * Set/Change a reference to the current parent classification via ID. EMPTY if there is no
-   * parent.
+   * Sets/Changes a reference to the current parent Classification via id. If the Classification has
+   * no parent, the value of parentId should be an empty String.
    *
-   * @param parentId The ID of the parent classification.
+   * @param parentId the id of the parent Classification
    */
   void setParentId(String parentId);
 
   /**
-   * Set/Change a reference to the current parent classification via key. EMPTY if there is no
-   * parent.
+   * Sets/Changes a reference to the current parent Classification via key. If the Classification
+   * has no parent, the value of parentKey should be an empty String.
```

Comment:
I understand why we don't link the class itself. Does this count for its methods aswell? Does our JavaDoc guide cover this case?

Reply:
The JavaDoc guide doesn't cover this case explicitly. Should we link them? They are not linked anywhere currently.

Reply:
-> separate ticket. I have wrote it down